__tests__/items-container.test.tsx

```diff
@@ -334,71 +711,244 @@ describe('ItemsContainer component behaviour', () => {
     });
-    
+
     test('load method calls manywho.engine.fileDataRequest if model.fileDataRequest is present', () => {
         componentWrapper = manyWhoMount({
             fileDataRequest: {},
         });
 
         componentWrapper.instance().load();
-        
+
         expect(globalAny.window.manywho.engine.fileDataRequest).toBeCalled();
     });
 
     test('load method forces update if model.objectDataRequest and model.fileDataRequest are falsy', () => {
         componentWrapper = manyWhoMount();
 
         const forceUpdateSpy = jest.spyOn(componentWrapper.instance(), 'forceUpdate');
-        
+
         componentWrapper.instance().load();
 
         expect(forceUpdateSpy).toBeCalled();
     });
 
     test('onNext calls onPaginate with next page number', () => {
         const onPaginateSpy = jest.spyOn(ItemsContainer.prototype, 'onPaginate');
-        
+
         componentWrapper = manyWhoMount();
 
         globalAny.window.manywho.state.getComponent = () => ({ page: 3 });
-        
+
         componentWrapper.instance().onNext();
 
         expect(onPaginateSpy).toBeCalledWith(4);
     });
 
     test('onPrev calls onPaginate with previous page number', () => {
         const onPaginateSpy = jest.spyOn(ItemsContainer.prototype, 'onPaginate');
-        
+
         componentWrapper = manyWhoMount();
 
         globalAny.window.manywho.state.getComponent = () => ({ page: 3 });
-        
+
         componentWrapper.instance().onPrev();
 
         expect(onPaginateSpy).toBeCalledWith(2);
     });
 
     test('onFirstPage calls onPaginate with 1', () => {
         const onPaginateSpy = jest.spyOn(ItemsContainer.prototype, 'onPaginate');
-        
+
         componentWrapper = manyWhoMount();
 
         globalAny.window.manywho.state.getComponent = () => ({ page: 3 });
-        
+
         componentWrapper.instance().onFirstPage();
 
         expect(onPaginateSpy).toBeCalledWith(1);
     });
 
+    test('Check sorting empty array does not crash', () => {
+        componentWrapper = manyWhoMount();
+
+        const empty = [];
+        empty.sort(componentWrapper.instance().compare('', true));
+        expect(empty.length).toBe(0);
+
+        empty.sort(componentWrapper.instance().compare('', false));
+        expect(empty.length).toBe(0);
+    });
+
+    test('Check sorting a single item does nothing', () => {
+        componentWrapper = manyWhoMount();
+
+        const single = [
+            {
+                developerName: 'OneRow',
+                isSelected: false,
+                order: 0,
+                properties: [
+                    {
+                        contentFormat: '',
+                        contentType: 'ContentString',
+                        contentValue: 'my value',
+                        developerName: 'stuff',
+                        objectData: null,
+                    },
+                ],
+            },
+        ];
+        const objectData = Object.assign(single);
```

Comment:
I don't think these Object.assigns are needed in the end.
Just an = seems fine.

Reply:
= just copies the reference, Object.assign is correct but as you pointed out, I'm doing it wrong. Will change as discussed to use correctly.